lib/checkPullRequestTemplate.js

```diff
@@ -0,0 +1,187 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Handler to check that the pull request template is followed.
+ */
+
+const { swapCharInPlace } = require('./utils');
+
+/**
+ * checks that the pull request's body contains the explanation.
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+const checkPRExplanation = (pullRequest) => {
+  // Restrict search to the overview section.
+  const overviewText = '## Overview';
+  const overviewHeadingIndex = pullRequest.body.indexOf(overviewText);
+
+  // Checks that there is an overview.
+  if (overviewHeadingIndex !== -1) {
```

Comment:
Let us valid and invalid examples as comments for all the checks - will make the code more readable.

Reply:
I'm not sure I understand what you mean.
Do you mean I should add a comment that shows what the template would look like in each check?

Reply:
Yes, similar to the ones you have in test files.

Reply:
Okay, done.